content/topics/bootcamp/bootcamp-android-enviroment-setup/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+Title: Bootcamp Mobile Environment Setup
+ready: true
+---
+
+We recommend Spck Code Editor
```

Comment:
What if people have computers? It would be good to say why we recommend this, and for who. 

Try empathise with the clueless noobs who will be here. 
Consider: https://www.osnews.com/story/19266/wtfsm/ 

I'd recommend recommending vscode